server/gabriel/common/log.py

```diff
@@ -25,30 +25,39 @@
 from .config import Const, Debug
 
 loggers = dict()
-DEFAULT_FORMATTER = '%(asctime)s %(name)s %(levelname)s %(message)s'
```

Comment:
This variable is not used anywhere in the gabriel repository. getLogger offers no way to specify the format, so I don't think there is any reason to keep this around.